README.md

```diff
@@ -1,3 +1,103 @@
 # ecomm
 
-Projeto de Ecommerce criando durante o programa LevelUp da Alura
\ No newline at end of file
+Projeto de Ecommerce criando durante o programa LevelUp da Alura
+
+# Os Doze Fatores
+
+## I. Base de Código :heavy_check_mark:
+
+<em>Uma base de código com rastreamento utilizando controle de revisão, muitos deploys</em> 
```

Comment:
Tente utilizar as marcações do Markdown ao invés de HTML - https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax